### ISSUE TITLE
Website links to Japanese car-buying site

### ISSUE THREAD

**Issue by halfnibble** [blackberry/NME#5]
The website, www.nme.io, goes to a Japanese, car-buying blog.  :,(

**Comment by jgranick** [comment]
We should update this to refer to https://www.openfl.org, somehow

**Comment by halfnibble** [comment]
@jgranick Yes. That would be nice. I wonder if anyone still works at BlackBerry? I regularly browse their GitHub to see if I can determine what BBRY is up to--so I can determine whether or not I should hold onto their stock.
